udata/routing.py

```diff
@@ -86,24 +86,25 @@ def quote(self, value):
             return url_quote(value)
 
     def to_python(self, value):
+        try:
```

Comment:
Change the order slug/id/404 to id/slug/404 in the parent function documentation